docs/buildah-commit.md

```diff
@@ -13,10 +13,17 @@ specified, an ID is assigned, but no name is assigned to the image.
 
 ## OPTIONS
 
+**-c**, **--change**=[]
+   Apply specified Dockerfile instructions while committing the image
+   Supported Dockerfile instructions: `CMD`|`ENTRYPOINT`|`ENV`|`EXPOSE`|`ONBUILD`|`USER`|`VOLUME`|`WORKDIR`
```

Comment:
Looks like you're supporting: 
ARCH, CMD, CREATED-BY, ENTRYPOINT, ENV, EXPOSE, LABEL, MAINTAINTER, OS, USER, VOLUME, WORKDIR in the code, IDK if you wanted to keep a few on the down low or not.